src/__tests__/__snapshots__/OpeningDate.test.js.snap

```diff
@@ -1450,7 +1450,7 @@ exports[`<OpeningDate /> matches the previous snapshot 1`] = `
               <span
                 className="emotion-43"
               >
-                For example, 15 08 2018
+                For example, 18 08 2018
```

Comment:
Is this going to be another snapshot that updates daily?

If yes, can we just remove the snapshot test for now until we decide what we're doing with snapshots across the board?

Reply:
I think this is an old one